bin/bench

```diff
@@ -3,16 +3,32 @@
 const path = require('path');
 const program = require('commander');
 const run = require('../lib/run');
+const mkdirp = require('mkdirp').sync;
+const { dirname, readFileSync, writeFileSync } = require('fs');
 
 let input = path.resolve('bench');
 
 program
   .version('0.1.0')
   .arguments('<directory>')
   .option('-i, --iterations <n>', 'The number of iterations to run', parseInt)
-  .action((directory) => input = path.resolve(directory))
-  .parse(process.argv);
+  .action((directory) => {
+    let iterations = program.iterations || 1;
+    directory = path.resolve(directory) || input;
+    run(directory, iterations);
+  });
 
-let iterations = program.iterations || 1;
+program
+  .command('generate')
+  .alias('g')
+  .description('Generates a new benchmark setup')
+  .option('-d, --directory <directory>', 'Directory for benchmarks')
+  .option('-n, --name <name>', 'The name of the benchmark')
```

Comment:
I think `name` should be a plain `arg` so that those with a default setup can just do `bench g some-benchmark`.